src/store/selectors/firestore.ts

```diff
@@ -0,0 +1,3 @@
+import { RootState } from 'app/store';
+
+export const getData = (state: RootState) => state.firestore.data;
```

Comment:
it's a bit vague. What type of Data is it? Maybe let's refactor it to `getMessages`? and avoid `data` as variable name in general

Reply:
Also, we should not group redux states by the source (`firestore`) but by the purpose (`messages`)